docs/format/source/software_process.rst

```diff
@@ -0,0 +1,28 @@
+The "dev" Branch and Releases
+=============================
+
+The default branch is "dev". The "dev" branch will hold the bleeding edge version of the NWB format specification,
+language specification, and storage specification. PRs should be made to "dev". Every PR should include an update to
+the corresponding format/language/storage release notes. If the PR involves a change to the NWB format, the PR should
+also update the version of the format in three places: `/docs/format/conf.py`, `/core/nwb.namespace.yaml` and
+`/core/nwb.file.yaml`. The new version string should be the next bugfix/minor/major version of the format with the
+suffix "-alpha". For example, if the format is currently in version "2.2.0" and the format is then updated with a bug
```

Comment:
This is inconsistent with https://nwb-extensions.github.io/versioning_guidelines  I think we should just use the ``a``, ``b``, and ``rc`` version tags from the versionining guidelines rather than ``-alpha``.

Reply:
I changed the text to be consistent with the extension versioning guidelines. Let me know what you think.